README.md

```diff
@@ -1 +1,18 @@
-# aws-lambda
\ No newline at end of file
+# aws-lambda
+
+author Tahmina Ringer
+
+## Links/Resources
+
+- [GitHub PR](https://github.com/Tahmina-Ringer-401-advanced-javascript/bearer-auth/pull/2)
+- [AWS](https://s3.console.aws.amazon.com/s3/home?region=us-west-2)
+
+- Resource[AWS](https://docs.aws.amazon.com/lambda/latest/dg/with-s3-example.html)
+
+Created an app where the user should be able to upload an image at any size, and have both the original size and a thumbnail size
+
+## Run
+
+- Upload a jpg to the lambda-lab17 lambda file and it will be resized and placed into the lambda-lab17-resize
+
+- My collegues, TAs and I were not able to solve my bug. I am able to upload a photo but it does not show up in the resize file
```

Comment:
I am sorry that you are having a bug, if you have a chance to come back to this, I will regrade for you!